src/tlo/util.py

```diff
@@ -436,3 +437,30 @@ def get_person_id_to_inherit_from(child_id, mother_id, population_dataframe, rng
         return abs(mother_id)
     elif mother_id >= 0:
         return mother_id
+
+
+def read_csv_files(resource_filepath: Path, file_name: list[str] = None) -> DataFrame | dict[str, DataFrame]:
```

Comment:
I would recommend not using the phrase "resource file path" for this as we use that for the destination of the overall resources folder. Instead how about the kwargs `folder: Path, files: list[str]`